src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -66,4 +73,8 @@ public UserTenantCollection getByUsernameAndTenantId(String username, String ten
     return result;
   }
 
+  private ConsortiumEntity checkConsortiumExistsOrThrow(UUID consortiumId) {
+    return consortiumRepository.findById(consortiumId).orElseThrow(() -> new ResourceNotFoundException("id", String.valueOf(consortiumId)));
```

Comment:
should "id" here be "consortiumId"? Since it is not referring to the association Id?

Reply:
fixed